### DISCUSSION TITLE
Hugmynd að bætingum við glósur.

### DISCUSSION THREAD

**Discussion by einargeir11** [HI-IDN/IDN302G-GitBook#6] [Hugmyndir]
Eins og flestir kannski muna að í upphafi áfangans voru ýmis vesen hjá nemendum við uppsetningu forritunarumhverfa, GitHub og annað slíkt. Með því sögðu, þá legg ég til að settur verður upp kafli sem miðar að "Algengum vandamálum" og "Algengum spurningum". Ég geri sterklega ráð fyrir því að þetta muni létta vinnu kennarans hvað varðar tæknileg vandamál ef kennarinn minnir nemendur á þessa kafla í glósunum. Einnig gæti þá kennarinn lagt meiri tíma í kennslustundum að hjálpa nemendum með verkefnin.

**Comment by tungufoss** [comment]
Takk fyrir ábendinguna! Tekin áfram í issue #54 svo hægt sé að vinna úr henni. Loka þessari umræðu sem afgreiddri.
